defaults/main.yml

```diff
@@ -92,6 +92,10 @@ libvirt_vm_virsh_default_env: "{{  { 'LIBVIRT_DEFAULT_URI': libvirt_vm_uri } if
 # Override for the libvirt connection uri. Leave unset to use the default.
 libvirt_vm_uri: ""
 
+# Optionally avoid sudo commands, this can be used with qemu:///session
+# to create VMs as an unprivileged user
+libvirt_vm_nosudo: false
```

Comment:
nit: would prefer if it was libvirt_vm_sudo: true

Reply:
done